app/helpers/inventories_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module InventoriesHelper
+  def inventory_shared_status_icon(inventory, team)
+    if inventory.shared_with?(team)
+      if inventory.shared_with_write?(team)
+        draw_custom_icon('shared-edit')
+      else
+        draw_custom_icon('shared-read')
+      end
+    elsif inventory.shared_with_anybody?
+      draw_custom_icon('i-shared')
```

Comment:
For ability to dynamic change this icon, when we share/unshare inventory. We must draw it everytime, if it is not shared inventory. We need only swap `hidden` class.

```
<span class="repository-share-status <%= "hidden" unless @repository.i_shared?(current_team) %>"  >	
          <%= draw_custom_icon('i-shared') %>	
</span>
```